app/models/drift_state/purging.rb

```diff
@@ -4,14 +4,14 @@ module DriftState::Purging
 
   module ClassMethods
     def purge_mode_and_value
-      value = VMDB::Config.new("vmdb").config.fetch_path(:drift_states, :history, :keep_drift_states)
+      value = ::Settings.fetch_path(:drift_states, :history, :keep_drift_states)
       mode  = (value.nil? || value.number_with_method?) ? :date : :remaining
       value = (value || 6.months).to_i_with_method.seconds.ago.utc if mode == :date
       return mode, value
     end
 
     def purge_window_size
-      VMDB::Config.new("vmdb").config.fetch_path(:drift_states, :history, :purge_window_size) || 10000
+      ::Settings.fetch_path(:drift_states, :history, :purge_window_size) || 10_000
```

Comment:
No need for defaultsin the source anymore...please move the default to settngs.yml directly.